<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRank</class>
 <widget class="QDialog" name="DialogRank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>龙虎榜</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>初级：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Begin">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>中级：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Intermediate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>高级：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Senior">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogRank</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
